frontend/rust-lib/flowy-database2/src/services/field/type_options/type_option.rs

```diff
@@ -87,14 +88,14 @@ pub trait TypeOptionCellDataSerde: TypeOption {
   fn parse_cell(&self, cell: &Cell) -> FlowyResult<<Self as TypeOption>::CellData>;
 }
 
-/// This trait that provides methods to extend the [TypeOption::CellData] functionalities.
-pub trait TypeOptionCellData {
-  /// Checks if the cell content is considered empty based on certain criteria. e.g. empty text,
-  /// no date selected, no selected options
-  fn is_cell_empty(&self) -> bool {
-    false
-  }
-}
+// /// This trait that provides methods to extend the [TypeOption::CellData] functionalities.
+// pub trait TypeOptionCellData {
+//   /// Checks if the cell content is considered empty based on certain criteria. e.g. empty text,
+//   /// no date selected, no selected options
+//   fn is_empty(&self) -> bool {
+//     false
+//   }
+// }
 
```

Comment:
If this is moved to collab, can we remove it from here

Reply:
Can